src/lib/list/list.md

```diff
@@ -42,6 +42,18 @@ element in an `<mat-list-item>`.
 </mat-nav-list>
 ```
 
+### Action lists
+
+Use `mat-action-list` tags for action lists (i.e. lists that have button tags).
+
+Simple action lists can use the `mat-list-item` attribute on button tag elements directly:
+
+```html
+<mat-action-list>
+  <button mat-list-item *ngFor="let link of links"> {{link.name}} </button>
+</mat-action-list>
```

Comment:
I would make the following changes:
```html
<mat-action-list>
  <button mat-list-item (click)="save()">Save</button>
  <button mat-list-item (click)="undo()">Undo</button>
</mat-action-list>
```
* Getting rid of `ngFor` makes it a bit simpler
* Adding a `(click)` handler shows how the action is performed